Update shopify_hydrogen.md (#402)
layer0-docs/layer0-docs: src/pages/guides/shopify_hydrogen.md

[Node.js](https://nodejs.org/en/) version 16.5.0 or higher

> _Note_: {{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version] for more information.

Author: ```suggestion
_Note_: {{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version) for more information.
```

> _Note_: {{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version) for more information.

Author: ```suggestion
_Note_: {{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version) guide for more information.
```

> _Note_: {{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version) guide for more information.

Author: ```suggestion
<Callout type="info">
{{ PRODUCT_NAME }} production runtime targets Node {{ NODE_VERSION }}. There may be some unexpected behavior building your project with a later version. See the [Node.js](install_nodejs#production-version) guide for more information.
</Callout>
```
